src/Commands/SiteStudioSettings.php

```diff
@@ -0,0 +1,225 @@
+      if ($status === SAVED_NEW) {
+        $this->io()->success($type . ' ' . $setting .  " is successfully created.");
+      }
+    }
+  }
+
+  /**
+   * Get data needed for site studio entity.
+   *
+   * @param array $data
+   *   Array of Data.
+   *
+   * @return array|null
+   *   Font stack data.
+   */
+  private function getSiteStudioData(array $data, $param) {
+    $site_studio_data = [];
+    foreach ($data as $item) {
+      // To allow existing entity update.
+      if (!$this->validateMachineName($item['machine_name'])) {
+        continue;
+      }
+      // Type value will distinguish between the cohesion entities.
+      // Command with param -> type = param.
+      // Command with no param -> type value from sheet data to be considered.
+      $type = $param ?: strtolower($item['type']);
+      switch ($type) {
+        case 'font':
+          // Prepare json values only if item has font data.
+          $json_values = ($item['type'] === 'Font') ? $this->getFontJsonValues($item) : [];
+
+          break;
+        case 'color':
+          // Prepare json values only if item has color data.
+          $json_values = ($item['type'] === 'Color') ? $this->getColorJsonValues($item) : [];
+
+          break;
+      }
+      // Site studio entity configuration.
+      if (!empty($json_values)) {
+        $entity = [
+          'langcode' => 'en',
+          'status' => true,
+          'label' => $item['label'],
+          'id' => $item['machine_name'],
+          'uuid' => $this->uuid->generate(),
+          'json_values' => json_encode($json_values),
+          'json_mapper' => '{}',
+          'modified' => true,
+          'selectable' => true,
+          'locked' => false,
+          'weight' => 0,
+        ];
+        $site_studio_data[] = [
+          'entity' => $entity,
+          'type' => $item['type'],
+        ];
+      }
+    }
+
+    return $site_studio_data;
+  }
+
+  /**
+   * Prepares json value for cohesion font stack.
+   *
+   * @param $item
+   *  The data coming from DST sheet.
+   * @return array
+   *  Returns json_values.
+   */
+  protected function getFontJsonValues($item): array {
+    return [
+      'name' => $item['label'],
+      'fontStack' => $item['metadata'],
+      'variable' => '$coh-' . $item['machine_name'],
+      'uid' => $item['machine_name'],
+      'class' => '.coh-' . $item['machine_name'],
+    ];
+
+  }
+
+  /**
+   * Prepares json value for cohesion color.
+   *
+   * @param $item
+   *  The data coming from DST sheet.
+   * @return array
+   *  Returns json_values.
+   */
+  protected function getColorJsonValues($item): array {
+    $tags = [];
+    $rgba_color = $this->hex2rgba($item['metadata'], 1);
+    if (array_key_exists('tag', $item) && $item['tag']) {
+      $tag = [$item['tag']];
+    } else {
+      $tag = [];
+    }
+    return [
+      'link' => true,
+      'value' => [
+        'value' => [
```

Comment:
is this duplicate `value` works fine OR this is the format required?

Reply:
This is the format of json_values for cohesion color.